<?xml version="1.0"?>
<launch>
	
	<include file="$(find gobot_software)/launch/robot_infos_params.launch"/>

	<node pkg="gobot_software" type="play_path" name="function_play_path" output="screen" respawn="true"/>
	<node pkg="gobot_software" type="auto_docking" name="function_auto_docking" output="screen" respawn="true" />
	<node pkg="gobot_software" type="battery_check" name="function_battery_check" output="screen" respawn="true" />
</launch>
